docs/explanation/architecture.md

```diff
@@ -57,41 +75,55 @@ All snap resources must be executed under the special snap user `_daemon_` only!
 
 ## Integrations
 
+The following charms can be integrated with PostgreSQL out of the box.  
+
 ### PgBouncer
 
-[PgBouncer](http://www.pgbouncer.org/) is a lightweight connection pooler for PostgreSQL that provides transparent routing between your application and back-end PostgreSQL Servers. The "[PgBouncer](https://charmhub.io/pgbouncer)" is an independent charm "Charmed PostgreSQL" can be related with.
+[PgBouncer](http://www.pgbouncer.org/) is a lightweight connection pooler for PostgreSQL that provides transparent routing between your application and back-end PostgreSQL Servers. 
 
 ### TLS certificates operator
 
-The [TLS Certificates](https://charmhub.io/tls-certificates-operator) charm is responsible for distributing certificates through relationship. Certificates are provided by the operator through Juju configs. For the playground deployments, the [self-signed operator](https://charmhub.io/self-signed-certificates) is available as well.
+The [TLS Certificates](https://charmhub.io/tls-certificates-operator) charm is responsible for distributing certificates through Juju configs. For test deployments, the [self-signed certificates](https://charmhub.io/self-signed-certificates) charm is available as well.
```

Comment:
please create followup backlo ticket to stop advertising deprecated tls-certificates-operator. self-signed-certificates should be used for the self-signed story and production cases are too different https://charmhub.io/topics/security-with-x-509-certificates#heading--question-2-what-type-of-certificates-do-you-need


I am not sure how to fit it here, this is why branching to followup ticket is necessary.